frontend/test/__support__/ui.tsx

```diff
@@ -59,14 +58,27 @@ export function renderWithProviders(
     initialState = _.pick(initialState, ...publicReducerNames) as State;
   }
 
+  const history = withRouter
+    ? createMemoryHistory({ entries: [initialRoute] })
+    : undefined;
+
   const reducers = mode === "default" ? mainReducers : publicReducers;
-  const store = getStore(reducers, initialState);
+
+  if (withRouter) {
+    Object.assign(reducers, { routing: routerReducer });
+  }
+
+  const store = getStore(
```

Comment:
It seems that the need to add `react-router-redux` manually comes from using not real `getStore` function. Can we use the one used by the app? https://github.com/metabase/metabase/blob/f9e6ad857baa9416be9e1ada7ee20284ffa90928/frontend/src/metabase/store.js#L28